docs/reference/migration/migrate_8_0/snapshots.asciidoc

```diff
@@ -11,3 +11,14 @@ The `repositories.fs.compress` setting has been removed.
 
 Instead use the repository specific `compress` setting to enable compression. See <<modules-snapshots>> for information
 on the `compress` setting.
+
+[float]
+==== Compression of meta data files is now default on
+
+Previously, the default value for `compress` was `false`. The default has been changed to `true`. This aligns the
+default with documentation.
```

Comment:
should we change the snapshot docs for 7.x and below to say that the default is false? We can then leave this sentence out.

Reply:
Removed that sentence in fd7d5fed2a34293edf4c9a772f73d9570971dcb5 and will do a follow-up PR against 7.x